### ISSUE TITLE
Not all aliases imported from old device

### ISSUE THREAD

**Issue by simontb** [thunderbird/thunderbird-android#7210]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.602

### Where did you get the app from?

Google Play

### Android version

13

### Device model

Fairphone 5

### Steps to reproduce

- Have an account with multiple aliases
- Some aliases don't have a name
- Use email address from alias without name as default sender address

Export the settings from old device and import to new one.

### Expected behavior

- All aliases are imported
- Default sender address is taken over

### Actual behavior

- Only aliases with name are imported
- Default sender address is first imported alias

**Comment by cketti** [comment]
In my test with K-9 Mail 6.602 identities without a name weren't exported. Can you please check your settings file to see if this indeed a problem with export, not import?

**Comment by simontb** [comment]
Sorry, I didn't distinguish between the two. I only looked at the export-import process E2E. From your reply I figured, that might've been misleading. My apologies!

Confirmed by looking at the export file that only identities with a name were exported. So definitely an export problem (can't test if import would've worked).

Default sender address and name seems not to be part of the export (at least couldn't spot it).
